README.md

```diff
@@ -1,11 +1,23 @@
 # Zeppelin Lab GmbH Styleguide
 This is the Z-Lab wide Styleguide project, to be used by all the ventures.
 
-# Usage
+## Usage
 
 Install it via npm:
 ```
 npm install github:Klickrent/zlab-style-guide
 ```
 
+## Adding new zamicons
+
+1. create a branch from master
+2. collect your svg images, one or more (be aware of naming for those images,
```

Comment:
(please use the naming conventions: minus (-) to split the words and only lowercase letters **kebab-case**